docs/public/static/diffs/template-bare-minimum/raw/45..50.diff

```diff
@@ -2500,10 +2500,10 @@ index 69e7eb9619..4cc4f0aaf5 100644
 -    "react-dom": "17.0.2",
 -    "react-native": "0.68.2",
 -    "react-native-web": "0.17.7"
-+    "expo": "~50.0.6",
++    "expo": "~50.0.11",
 +    "expo-status-bar": "~1.11.1",
 +    "react": "18.2.0",
-+    "react-native": "0.73.2"
++    "react-native": "0.73.4"
```

Comment:
should this be 0.73.5?

Reply:
Yeah, not sure why `et generate-bare-diffs` generated these changes

Reply:
Here you go https://github.com/expo/expo/pull/27520